sentry-samples/sentry-samples-spring-boot-jakarta/spy.log

```diff
@@ -0,0 +1,8 @@
+1665496251482|47|statement|connection 0|url jdbc:p6spy:hsqldb:mem:testdb|CREATE TABLE person ( id INTEGER IDENTITY PRIMARY KEY, firstName VARCHAR(50) NOT NULL, lastName VARCHAR(50) NOT NULL )|CREATE TABLE person ( id INTEGER IDENTITY PRIMARY KEY, firstName VARCHAR(50) NOT NULL, lastName VARCHAR(50) NOT NULL )
```

Comment:
Does this need to be in the source control?

Reply:
no, will remove